src/main/webapp/college-info.js

```diff
@@ -289,8 +289,61 @@ function createCommentElement(name, message, time) {
 function prepReviewForm(id) {
   const submitReviewForm = document.getElementById('submit-review');
   submitReviewForm.setAttribute('action', `/data?id=${id}`);
+
   const idInputElement = document.getElementById('school-id');
   idInputElement.setAttribute('value', id);
 }
 
+/**
+ * Sets the hidden elements to be shown and the shown elements to be hidden.
+ * @param {HTMLElement} showElement HTML element that is hidden/will be shown.
+ * @param {HTMLElement} hideElement HTML element that is shown/will be hidden.
+ * @param {HTMLElement} showElementNavButton HTML nav button to activate.
+ * @param {HTMLElement} hideElementNavButton HTML nav button to deactivate.
+ */
+function toggleElementsDisplay(showElement, hideElement, showElementNavButton,
+    hideElementNavButton) {
+  showElement.classList.remove('is-hidden');
+  hideElement.classList.add('is-hidden');
+
+  showElementNavButton.classList.add('active');
+  hideElementNavButton.classList.remove('active');
+}
+
+/**
+ * Sets the toggle functions to the nav bar buttons.
+ */
+function loadToggle() {
+  const infoDiv = document.getElementById('info');
+  const reviewsDiv = document.getElementById('reviews');
+  const infoNavButton = document.getElementById('info-nav');
+  const reviewsNavButton = document.getElementById('reviews-nav');
+
+  // Gets location hash from URL to show/hide appropriate section when the
+  // page initially loads.
+  if (location.hash == '#reviews') {
+    infoDiv.classList = 'is-hidden';
+  } else {
+    location.hash = 'info';
+    reviewsDiv.classList = 'is-hidden';
+    infoNavButton.classList.add('active');
+  }
+
+  // Adds toggle function to tab buttons.
+  infoNavButton.addEventListener('click',
+      () => {
+        toggleElementsDisplay(infoDiv, reviewsDiv,
+            infoNavButton, reviewsNavButton);
+
+        // Loads hash and refreshes page to show reload charts.
+        location.hash = 'info';
+        location.reload();
```

Comment:
Is it easy to do this only in the case where it's needed (after adding a new review)? It would be nice to keep the hack as localized as possible.

Reply:
I was trying that but the minimum time this is necessary is when you click back to info tab after having submitted a comment. However, the button and function for going back to the info tag is the same independent of whether or not a comment was submitted.

Furthermore, I also only keeping it to clicking the info tab and not loading the reviews tag however I was getting some race condition or something where it would go to the tab that was clicked but then reload back to the tab it was on instead of reloading on the new tab.

Reply:
Yeah, if it's not simple to work out, I think that's fine.